aws-redshift-cluster/src/main/java/software/amazon/redshift/cluster/BaseHandlerStd.java

```diff
@@ -207,7 +207,7 @@ protected boolean issueModifyClusterMaintenanceRequest(ResourceModel model) {
   // check for required parameters to not have null values
   protected boolean invalidCreateClusterRequest(ResourceModel model) {
     return model.getClusterIdentifier() == null || model.getNodeType() == null
-            || model.getMasterUsername() == null || model.getMasterUserPassword() == null;
+            || model.getMasterUsername() == null;
```

Comment:
if cx doesn't pass in ManageMasterPassword the request would still be invalid right, can we add the condition for all the scenarios here please? or tell me I'm wrong :)